bookwyrm/views/status.py

```diff
@@ -112,6 +112,18 @@ def post(self, request, status_type, existing_status_id=None):
         if status.reply_parent:
             status.mention_users.add(status.reply_parent.user)
 
+        # inspect the text for hashtags
+        for (mention_text, mention_hashtag) in find_hashtags(content).items():
+            # add them to status mentions fk
+            status.mention_hashtags.add(mention_hashtag)
+
+            # turn the mention into a link
+            content = re.sub(
+                rf"{mention_text}\b(?!@)",
+                rf'<a href="{mention_hashtag.remote_id}">{mention_text}</a>',
```

Comment:
We said in #1081 that an initial hashtag support doesn't need a page for listing the statuses mentioning a given hashtag yet. I find it weird not linking a hashtag though, as it gives no visual indication that hashtags are now a thing. Maybe this PR should include a fairly simple page listing maybe just all public statuses for a hashtag?

Reply:
I think that's a good idea! You're right that not linking a hashtag would be a weird choice, both for the visual queue and because once the hashtag page was created, older posts wouldn't link to it.

Reply:
Just playing around with adding a new view for this as we speak, I'll see shortly how hard it'll be to add one with respecting privacy settings of activities.

Reply:
Just added an initial version of such a page 👍